crates/oxc_allocator/src/lib.rs

```diff
@@ -20,6 +20,7 @@
 //!   Usage of this feature is not advisable, and it will be removed as soon as we're able to.
 
 #![warn(missing_docs)]
+#![cfg_attr(nightly, feature(allocator_api))]
```

Comment:
Now, we simply use `allocator_api` when we're on nightly. This is the correct usage of `allocator_api2` since version `0.3.0`.